pkg/channel/http/http_channel_test.go

```diff
@@ -68,7 +69,14 @@ func (t *testHandlerHeaders) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		headers[k] = v[0]
 	}
 	rsp, _ := json.Marshal(headers)
-	io.WriteString(w, string(rsp))
+
+	var rspBuffer bytes.Buffer
+	if _, err := rspBuffer.Write(rsp); err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		return
+	}
+
+	io.WriteString(w, rspBuffer.String())
```

Comment:
was this error check being highlighted with the newer linter?

Reply:
It was indeed as a staticcheck (SA6006 - Using io.WriteString to write []byte), I just clocked that I can just use the response writer - sleepy moment 😅